docs/commands/ft.create.md

```diff
@@ -24,7 +24,7 @@ Create an index with the given specification
     [NOFREQS] 
     [STOPWORDS count [stopword ...]] 
     [SKIPINITIALSCAN]
-    SCHEMA field_name [AS alias] TEXT | TAG | NUMERIC | GEO | VECTOR [ SORTABLE [UNF]] 
+    SCHEMA {identifier} AS {attribute} TEXT | TAG | NUMERIC | GEO | VECTOR [ SORTABLE [UNF]] 
```

Comment:
Since this is not generated, I suggest to split each type to a separate description with its relevant parameters, e.g., 

TEXT:       SORTABLE UNF  NOINDEX NOSTEM  WEIGHT PHONETIC WITHSUFFIXTRIE
TAG:         SORTABLE UNF  NOINDEX SEPARATOR CASESENSITIVE   WITHSUFFIXTRIE
NUMERIC:     SORTABLE NOINDEX
GEO:         SORTABLE NOINDEX
VECTOR - has a detailed dedicated page

Reply:
The above parameters are unordered except for `[SORTABLE [[UNF]]`

Reply:
@oshadmi does it make sense to add those syntax descriptions below where we discuss the options in a bullet list? Or you think we need to expand the syntax section?

Reply:
@nermiller It seems worthwhile to me to expand the syntax section with the appropriate parameters per attribute type.
Of course that would make the syntax section more populated / less concise.

If we detail the relevant parameters per attribute type where we discuss the options in the bullet list, users would need to get there even if they just want a peek at the syntax.

We can also consider just adding some place holder or link to the details below, e.g.,
SCHEMA {identifier} AS {attribute} TEXT | TAG | NUMERIC | GEO | VECTOR  **[see available parameters below...]**
Or just make the attribute types links to their relevant section below.

Currently it is partial and misleading.